docs/porkbun.md

```diff
@@ -30,6 +30,7 @@
 
 - `"ip_version"` can be `ipv4` (A records), or `ipv6` (AAAA records) or `ipv4 or ipv6` (update one of the two, depending on the public ip found). It defaults to `ipv4 or ipv6`.
 - `"ipv6_suffix"` is the IPv6 interface identifier suffix to use. It can be for example `0:0:0:0:72ad:8fbb:a54e:bedd/64`. If left empty, it defaults to no suffix and the raw public IPv6 address obtained is used in the record updating.
+- `"owner"` a CSV list of domains to update. For example, `"@,*"` for a domain of `domain.com` would update 2 DNS entries. `domain.com` and `*.domain.com`
```

Comment:
Did this just get fixed on the latest master since 2.7.0. I swear I tried CSV domain names, and it didn't work...
Just tried again and it's working just fine though, so idk what I did wrong. Maybe another reflection of the *.domain.tld issue as that really messed me up when I was trying to get this setup.